ibm/mas_devops/roles/mongodb/defaults/main.yml

```diff
@@ -75,7 +75,7 @@ ibmcloud_resourcegroup: "{{ lookup('env','IBMCLOUD_RESOURCEGROUP') | default('De
 ibmcloud_region: "{{ lookup('env', 'IBMCLOUD_REGION') | default('us-east', True) }}"
 ibmcloud_apikey: "{{ lookup('env', 'IBMCLOUD_APIKEY') }}"
 
-ibm_mongo_plan:  "{{ lookup('env', 'IBM_MONGO_PLAN') | default('standard',True) }}"
+ibm_mongo_plan: "{{ lookup('env', 'IBM_MONGO_PLAN') | default('standard',True) }}"
```

Comment:
Will fail ansible linting (when we introduce it) - no space after the comma `('standard',True)`